docs/conf.py

```diff
@@ -141,6 +142,7 @@
     r"https://pypi.org/manage/project/icalendar/collaboration/",
     # Ignore specific anchors
     r"https://github.com/collective/icalendar/blob/main/README.rst#related-projects",
+    r"https://www.unicode.org/cldr/cldr-aux/charts/29/supplemental/zone_tzid.html",
```

Comment:
Why do you want to ignore this link from the checker? It also lacks an anchor, which the comment above implies it should have.

```suggestion
```

Reply:
It breaks from time to time on github CI. I can move it up.